compose/ui/ui/src/webCommonW3C/kotlin/androidx/compose/ui/window/ComposeWindow.web.kt

```diff
@@ -462,6 +472,11 @@ internal class ComposeWindow(
         event: TouchEvent,
         offset: Offset,
     ) {
+        // iOS Safari doesn't request focus when the page is shown,
+        // and the lifecycle doesn't trigger ON_RESUME.
+        // so, we decided to handle every touch
+        lifecycle.handleLifecycleEvent(Lifecycle.Event.ON_RESUME)
```

Comment:
Could we have an `if (lifycecle.currentState != ON_RESUME) { handle on resume... }`?
So the handler code won't be invoked on every touch event which can happen frequently.

Reply:
there is already a check inside:
```
private fun moveToState(next: State) {
        if (state == next) {
            return
        }
```